Makefile

```diff
@@ -9,7 +9,6 @@ help: ## Print the help message
 .PHONY: install
 install:
 	poetry install
-	npm install
```

Comment:
We can fully remove this `make` target since if we're requiring Docker for the dev env

Reply:
I ended up pulling ruff/mypy out of this PR due to an error I encountered when running on CI